onmt/translate/decode_strategy.py

```diff
@@ -135,17 +138,18 @@ def initialize(self, enc_out, src_len, src_map=None, device=None,
         """
         if device is None:
             device = torch.device('cpu')
+        # Here we set the decoder to start with self.start (BOS or EOS)
         self.alive_seq = torch.full(
-            [self.batch_size * self.parallel_paths, 1], self.bos,
+            [self.batch_size * self.parallel_paths, 1], self.start,
             dtype=torch.long, device=device)
         self.is_finished = torch.zeros(
             [self.batch_size, self.parallel_paths],
             dtype=torch.uint8, device=device)
         if target_prefix is not None:
-            seq_len, batch_size, n_feats = target_prefix.size()
+            batch_size, seq_len, n_feats = target_prefix.size()
             assert batch_size == self.batch_size * self.parallel_paths,\
                 "forced target_prefix should've extend to same number of path!"
-            target_prefix_words = target_prefix[:, :, 0].transpose(0, 1)
+            target_prefix_words = target_prefix[:, :, 0]  # no features
```

Comment:
Not sure to understand this change // added comment.

Reply:
the transpose should have been removed when I switched batch and sequence order.
the comment is just to clarify we take the feature 0 only which "word".